modules/signals/rxjs-interop/src/rx-method.ts

```diff
@@ -30,23 +30,20 @@ export function rxMethod<Input>(
   destroyRef.onDestroy(() => sourceSub.unsubscribe());
 
   const rxMethodFn = (input: RxMethodInput<Input>) => {
-    if (isSignal(input)) {
-      const watcher = effect(() => source$.next(input()), { injector });
```

Comment:
Thanks for the quick catch Rainer!

No need to revert `rxMethod` implementation. It's enough to change this line to:

```ts
      const watcher = effect(() => {
        const value = input();
        untracked(() => source$.next(value));
      }, { injector });
```

and new tests will pass. 👌

Reply:
Yup, done! The test also passes. I removed the second test because I don't think it brings any additional benefit.